imagelab/DisplayImage.java

```diff
@@ -35,6 +36,11 @@
     private static int xspot = XMAX;
     /** y-coordinate for next window */
     private static int yspot = YMAX;
+    
+    /**Image Producer to set img*/
+    private MemoryImageSource source;
+    /**Used to get STD_DURATION*/
+    private Music music;
```

Comment:
If the only intent is to get STD_DURATION, why is instantiation needed?
Where is this used in the code?

Reply:
I have removed this object instantiation as STD_DURATION  was only used as a preliminary attempt for image synchronization. The constant is no longer used or needed for synchronization.